src/main/java/se/sundsvall/datawarehousereader/integration/stadsbacken/model/installedbase/InstalledBaseItemEntity.java

```diff
@@ -62,6 +62,9 @@ public class InstalledBaseItemEntity {
 	@Column(name = "DateTo", insertable = false, updatable = false, columnDefinition = "datetime")
 	private LocalDate dateTo;
 
+	@Column(name = "PlacementStatusModified", insertable = false, updatable = false, columnDefinition = "varchar(50)")
```

Comment:
`varchar(50)` 

This is weird. We should do something about it. Have you talked to the creator of this abomination?

Reply:
I've assigned our most fearsome watchdog on this (Jakob) but he has not gotten any response yet. But yes, it is weird to type a column containing dates as a text column.

Reply:
![image](https://github.com/Sundsvallskommun/api-service-datawarehousereader/assets/46994590/8c132de0-7bb7-4846-966a-32beb00bddf8)